website/src/Pages/ResourcesPage/AddResourcePage.jsx

```diff
@@ -0,0 +1,243 @@
+          <Select
+            value={tag}
+            onChange={(e) => {
+              setTag(e.target.value);
+            }}
+          >
+            <MenuItem value="food">food</MenuItem>
+            <MenuItem value="shelter">shelter</MenuItem>
+            <MenuItem value="health">health</MenuItem>
+            <MenuItem value="legal services">legal services</MenuItem>
+            <MenuItem value="shower">shower</MenuItem>
+            <MenuItem value="mission">mission</MenuItem>
+            <MenuItem value="social services">social services</MenuItem>
+          </Select>
+        </FormControl>
+      </Box>
+      <Box sx={{ display: 'flex' }}>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: days[0] ? '#4C4C9B' : '#D0D0E8' }}
+          onClick={() => updateDays(0)}
+        >
+          M
+        </Button>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: days[1] ? '#4C4C9B' : '#D0D0E8' }}
+          onClick={() => updateDays(1)}
+        >
+          Tu
+        </Button>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: days[2] ? '#4C4C9B' : '#D0D0E8' }}
+          onClick={() => updateDays(2)}
+        >
+          W
+        </Button>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: days[3] ? '#4C4C9B' : '#D0D0E8' }}
+          onClick={() => updateDays(3)}
+        >
+          Th
+        </Button>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: days[4] ? '#4C4C9B' : '#D0D0E8' }}
+          onClick={() => updateDays(4)}
+        >
+          F
+        </Button>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: days[5] ? '#4C4C9B' : '#D0D0E8' }}
+          onClick={() => updateDays(5)}
+        >
+          Sa
+        </Button>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: days[6] ? '#4C4C9B' : '#D0D0E8' }}
+          onClick={() => updateDays(6)}
+        >
+          Su
+        </Button>
+      </Box>
+      <Box>
+        <Typography>Phone Number</Typography>
+        <TextField
+          value={phoneNumber}
+          onChange={handlePhoneNumberChange}
+        />
+      </Box>
+      <Box>
+        <Typography>Hours</Typography>
+        <Box>
+          <LocalizationProvider dateAdapter={AdapterDayjs}>
+            <TimePicker
+              value={startTime}
+              onChange={(newTime) => setStartTime(newTime)}
+            />
+          </LocalizationProvider>
+        </Box>
+        <Typography>to</Typography>
+        <Box>
+          <LocalizationProvider dateAdapter={AdapterDayjs}>
+            <TimePicker
+              value={endTime}
+              onChange={(newTime) => setEndTime(newTime)}
+            />
+          </LocalizationProvider>
+        </Box>
+      </Box>
+      <Box>
+        <Box>
+          <Typography>Latitude</Typography>
```

Comment:
So we separate Latitude, Longitude, and Address fields, which may lead to collisions if inputs don't line up for some reason. I think it would be a good idea to obscure the coordinates stuff from the front-end, and [Google seems to have an API for it](https://developers.google.com/maps/documentation/geocoding/overview). Completely up to you, as this would entail some changes for the "reverse geo-coding" (i.e., deriving the coordinates from the address), which we would need to populate the latitude and longitude coordinates in our database (we would naturally need to perform some validation to make sure 1) the address is valid and 2) it is indeed the correct address the admins want; this could be done perhaps with a confirmation page that shows a map)

TLDR I think having Lat/Long/Addr fields is, at best, inconvenient for the admins and, at worst, a bit risky if admins make a mistake with the inputs

Reply:
I agree with the reasoning but because of time restrictions I think it's fine for this process. In the future improvements, it would be better to do all the checking to make the entire admin work easier